### PR TITLE
Automated benchmark

### PR DESCRIPTION
**Pull request by BaBrixx**

### CODE REVIEW

File: tests/benchmark/top-1000-benchmark.test.ts [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+			// Disable Plugins
+			for (const plugin of plugins) {
+				const checkbox = await popup.getByLabel(plugin.name);
+				await checkbox.setChecked(plugin.name == selectedPlugin.name);
+			}
+
+			// Click the "Scan Now" button
+			await popup.getByRole("button", { name: "Scan Now" }).click();
+
+			// Click the "Continue" button - if it exists
```
Comment (DenFlyvendeGed) [inline tests/benchmark/top-1000-benchmark.test.ts:52]: Den finder den vel kun? og "clicker" ikke på den

File: tests/benchmark/top-1000-benchmark.test.ts [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+			const continueButton = await popup.getByRole("button", {
+				name: "Continue",
+			});
+
+			if (selectedPlugin.continueShown) {
+				// Wait for the "Continue" button to be visible, with a timeout of 5 seconds
```
Comment (DenFlyvendeGed) [inline tests/benchmark/top-1000-benchmark.test.ts:58]:
```suggestion
				// Wait for the "Continue" button to be visible, with a timeout of 10 seconds
```

**Review by atjn — CHANGES_REQUESTED** [review]
There is no description of how to run/what it tests anywhere (might be nice in the readme), trying to run `npm run test:benchmark` fails becuase it tries to read `timeBeforeFirstResult.csv`, which does not exist.

**Review by atjn — APPROVED** [review]

**Review by atjn — APPROVED** [review]

File: tests/benchmark/result_timeBeforeFirstResult.csv [hunk 3]
```diff
@@ -0,0 +1,2085 @@
...
+fbsbx.com,User preference,0.044
+classlink.com,Format,0.092
+classlink.com,Text compression,0.089
+classlink.com,User preference,0.066
+aniview.com,Format,0.201
+aniview.com,Text compression,0.086
+aniview.com,User preference,0.216
+pornhub.com,Format,0.195
+pornhub.com,Text compression,0.051
+pornhub.com,User preference,0.115
```
Comment (NybyDK) [inline tests/benchmark/result_timeBeforeFirstResult.csv:1713]: YOU KINKY KINKY BOY

**Pull request merged**